Documentation/ceph-upgrade.md

```diff
@@ -319,21 +318,37 @@ spec:
 # ....
 ```
 
+### 7. Update Rook-Ceph custom resource definitions
+**IMPORTANT: Do not perform this step until ALL existing Rook-Ceph clusters are updated**
+
+After all Rook-Ceph clusters have been updated following the steps above, update the Rook-Ceph
+Custom Resource Definitions. This is important to get the latest schema validations that will
+support running external Ceph clusters and will also help with creating or modifying Rook-Ceph
+deployments in the future.
+
+```sh
+kubectl apply -f upgrade-from-v1.0-crds.yaml
+```
+
 # Ceph Version Upgrades
 Rook 1.0 was the last Rook release which will support Ceph's Luminous (v12.x.x) version. Users are
-required to upgrade to Mimic (v13.2.4 or newer) or Nautilus (v14.2.x) now. Rook 1.1 will only run with
-Mimic or newer, though running with Octopus requires the `allowUnsupported: true` flag.
-
-**IMPORTANT: This section only applies to clusters running Rook 1.0 or newer**
+required to upgrade to Mimic (v13.2.4 or newer) or Nautilus (v14.2.x) now. These are the only
```

Comment:
Is there a comment about this earlier in the doc? It should be a pre-requisite before updating to v1.1. This is a bit late to mention it.

Reply:
That depends on if Rook can upgrade the Rook and Ceph version at the same time right?

Reply:
I don't see how Rook and Ceph could be updated at the same time without doing something disruptive like restart the operator. Rook upgrades are triggered by updating the operator image and Ceph upgrades are triggered with a new ceph/ceph image in the cluster CR. Since they are separate, if they are on Luminous they would need to update Ceph before updating to Rook 1.1 or else the operator will just fail in the 1.1 upgrade.